pkg/remotewrite/config.go

```diff
@@ -41,6 +41,16 @@ type Config struct {
 	// Password is the Password for the Basic Auth.
 	Password null.String `json:"password"`
 
+	// ClientCertificate is the public key of the SSL certificate.
+	// It is expected the path of the certificate on the file system.
+	// If it is required a dedicated Certifacate Authority then it should be added
```

Comment:
It theory we could also provide the way to add the CA like it's implemented in k6, but not sure if it's under the scope of this PR

Reply:
I would prefer if we get an explicit request from users for it. The demand for `mTLS` feature seems to be not so high in general.